src/containers/ChainButton/ChainButton.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Button as ButtonComponent } from 'decentraland-ui'
+import { Props } from './ChainButton.types'
+import ChainCheck from '../ChainCheck'
+
+const disabledStyle = { opacity: 0.5, cursor: 'not-allowed', transform: 'none' }
```

Comment:
We don't have CSS files on `decentraland-dapps`, and the current disabled styles of the `Button` don't let the `Popup` button work, because of the `pointer-events: none` rule.

Reply:
Oh, that's right, we don't have a CSS processor here.